gcloud-java-storage/src/main/java/com/google/gcloud/spi/StorageRpc.java

```diff
@@ -42,6 +42,8 @@
     IF_SOURCE_METAGENERATION_NOT_MATCH("ifSourceMetagenerationNotMatch"),
     IF_SOURCE_GENERATION_MATCH("ifSourceGenerationMatch"),
     IF_SOURCE_GENERATION_NOT_MATCH("ifSourceGenerationNotMatch"),
+    IF_MD5_MATCH("md5Hash"),
```

Comment:
See my comment about avoid a change in the RPC layer but if we must then I would suggest naming them here as INCLUDE_MD5 and INCLUDE_CRC32C.